vendor.go

```diff
@@ -0,0 +1,13 @@
+// +build vendor
+
+package main
+
+// This file exists to trick "go mod vendor" to include "main" packages.
+// It is not expected to build, the build tag above is only to prevent this
+// file from being included in builds.
+
+import (
+	_ "k8s.io/code-generator"
```

Comment:
minor: I would suggest to put into some pkg build util.go etc rather than a separate vendor file outside.